deploy/role.yaml

```diff
@@ -57,132 +57,7 @@ kind: ClusterRole
 metadata:
   name: knative-serving-operator
-  labels:
-    serving.knative.dev/release: devel
-    serving.knative.dev/controller: "true"
 rules:
-  - apiGroups: [""]
-    resources: ["pods", "namespaces", "secrets", "configmaps", "endpoints", "services", "events", "serviceaccounts"]
-    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
-  - apiGroups: ["extensions"]
-    resources: ["ingresses","deployments"]
-    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
-  - apiGroups: ["apps"]
-    resources: ["deployments", "deployments/scale", "statefulsets"]
-    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
-  - apiGroups: ["admissionregistration.k8s.io"]
-    resources: ["mutatingwebhookconfigurations"]
-    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
-  - apiGroups: ["apiextensions.k8s.io"]
-    resources: ["customresourcedefinitions"]
-    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
-  - apiGroups: ["serving.knative.dev"]
-    resources: ["configurations", "routes", "revisions", "services"]
-    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
-  - apiGroups: ["serving.knative.dev"]
-    resources: ["configurations/status", "routes/status", "revisions/status", "services/status"]
-    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
-  - apiGroups: ["autoscaling.internal.knative.dev"]
-    resources: ["podautoscalers", "podautoscalers/status"]
-    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
-  - apiGroups: ["autoscaling"]
-    resources: ["horizontalpodautoscalers"]
-    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
-  - apiGroups: ["caching.internal.knative.dev"]
-    resources: ["images"]
-    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
-  - apiGroups: ["networking.internal.knative.dev"]
-    resources: ["clusteringresses", "clusteringresses/status", "serverlessservices", "serverlessservices/status"]
-    verbs: ["get", "list", "create", "update", "delete", "deletecollection", "patch", "watch"]
-  - apiGroups: ["networking.istio.io"]
-    resources: ["virtualservices", "gateways"]
-    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
-
-  # Rules handled by aggregation labels upstream
-
-  - apiGroups:
-    - build.knative.dev
-    resources:
-    - builds
-    verbs:
-    - get
-    - list
-    - create
-    - update
-    - delete
-    - patch
-    - watch
-  - apiGroups:
-    - pipeline.knative.dev
-    resources:
-    - pipelines
-    - tasks
-    - pipelineresources
-    - pipelineruns
-    - taskruns
-    verbs:
-    - get
-    - list
-    - create
-    - update
-    - delete
-    - patch
-    - watch
-
-  # The above rules are from upstream. The remaining are
-  # required for OpenShift
-
-  - apiGroups:
-    - security.openshift.io
-    resources:
```

Comment:
remove these too ?

Reply:
I'd like to get them upstream, if possible. Otherwise, they should really be post-edits on the upstream yaml in `deploy/resources` if we need them.